tests/api/test_BitMask.py

```diff
@@ -150,13 +150,41 @@ def test_subset_a_list(self):
                 mask.set_kth_bit(index)
 
         all_ones = mask.masked_select_from_list(list_)
-        mask.x = mask.x ^ (2 ** n - 1) # inverse
+        mask.invert(n)
         all_zeros = mask.masked_select_from_list(list_)
         self.assertGreater(len(all_ones), 0)
         self.assertGreater(len(all_zeros), 0)
         self.assertTrue(all([item_ > 0 for item_ in all_ones]))
         self.assertTrue(all([item_ == 0 for item_ in all_zeros]))
 
+
+    def test_masked_select_from_list_example(self):
+        list_ = [1, 2, 3, 4, 5, 6]
+        mask = BitMask.from_bin('0b001101') # expected result is [1, 3, 4]
```

Comment:
still feels weird that it should be correct to be right to left.. I need to check this in typescript if we both are doing the same :D

Reply:
I wrote more tests on the typescript side and we dot it the same way from right to left. Makes sense